### ISSUE TITLE
Snippet expansion is not working when editing a file through canvas

### ISSUE THREAD

**Issue by Yyote** [ArianaKhit/text-snippets-obsidian#59]
Steps to repeat the problem:
1. Create obsidian canvas
2. Add an existing note to the canvas
3. Edit the note and try to expand any snippet

Expected result: expansion works as always.
Actual result: nothing happens.

**Comment by pnchinmay** [comment]
Check your hotkeys
Ctrl+Tab might be binded to something else
I had the same issue

**Comment by Yyote** [comment]
> Check your hotkeys Ctrl+Tab might be binded to something else I had the same issue

Nope, it's only used with snippets. Also tried to use Hotkey Chords plugin, didn't help too. To be particular, I am using Shift+Space as a hotkey.

**Comment by pnchinmay** [comment]
> > Check your hotkeys Ctrl+Tab might be binded to something else I had the same issue
> 
> Nope, it's only used with snippets. Also tried to use Hotkey Chords plugin, didn't help too. To be particular, I am using Shift+Space as a hotkey.

yeah only ctrl+Tab is working
I've earlier tried with different keys. It didn't work

**Comment by Yyote** [comment]
> > > Check your hotkeys Ctrl+Tab might be binded to something else I had the same issue
> > 
> > 
> > Nope, it's only used with snippets. Also tried to use Hotkey Chords plugin, didn't help too. To be particular, I am using Shift+Space as a hotkey.
> 
> yeah only ctrl+Tab is working I've earlier tried with different keys. It didn't work

Unfortunately, it doesn't work too. Tried changing back to Ctrl+Tab - didn't work. Reloaded - didn't work.
